apps/corona-banner/src/app.js

```diff
@@ -1,52 +1,96 @@
-import React, { useState, useEffect } from "react";
-import CoronaSvg from "./assets/covid-virus-icon.svg";
-import Chevron from "../../../images/chevron.svg";
+import React, { useState, useEffect } from 'react'
+import axios from 'axios'
+import CoronaSvg from './assets/covid-virus-icon.svg'
+import Chevron from '../../../images/chevron.svg'
+
+const Banner = ({
+  newCases,
+  hospitalised,
+  deaths,
+  vaccinated,
+  vaccinatedPercentage,
+}) => {
+
+  return (
+    <a href='/tagg/coronaviruset/'>
```

Comment:
This will lead to `frontends.ksfmedia.fi/corona-banner/tagg/coronaviruset` which is probably not intended 🙂